### PR TITLE
flatpak-builder: Allow specifying CPU arch

### LINKED ISSUE #35: Arch option / cross-compile
Is it possible to add an option for `arch` so that we can cross-compile for eg. `aarch64`? It would be convenient to have builds churned out for x86_64 and aarch64 for testing on mobile devices.

### PR DESCRIPTION
**Pull request by davidmhewitt**
Fixes #35

Adds support for specifying the `--arch` argument to flatpak-builder and adds a guide to the README about how to use it (i.e. if building ARM, you either need an self-hosted ARM builder, or to enable emulation with QEMU, which is documented)

We're likely going to need this in elementary soon so that we can re-enable our Raspberry Pi and Pinebook Pro builds.

Please let me know any feedback!

Thanks!

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
Overall looks good to me, thanks ! Do you mind modifying this build test https://github.com/bilelmoussaoui/flatpak-github-actions/blob/master/.github/workflows/flatpak-test.yml to build against two different architectures?

**Comment by davidmhewitt** [comment]
@bilelmoussaoui Yes, of course! That should be done now.

File: README.md [hunk 1]
```diff
@@ -48,6 +48,50 @@ jobs:
 | `branch` | The default flatpak branch  | Optional | `master` |
 | `cache` | Enable/Disable caching `.flatpak-builder` directory | Optional | `true` |
 | `cache-key` | Specifies the cache key | Optional | `flatpak-builder-${sha256(manifestPath)}` |
+| `arch` | Specifies the CPU architecture to build for | Optional | `x86_64` |
+
+#### Building for multiple CPU architectures
+
+To build for CPU architectures other than `x86_64`, the GitHub Actions workflow has to either natively be running on that architecture (e.g. on an `aarch64` self-hosted GitHub Actions runner), or the container used must be configured to emulate the requested architecture (e.g. with QEMU).
+
+For example, to built a Flatpak for both `x86_64` and `aarch64` using emulation, use the following workflow as a guide:
```
Comment (bilelmoussaoui) [inline README.md:57]:
```suggestion
For example, to build a Flatpak for both `x86_64` and `aarch64` using emulation, use the following workflow as a guide:
```

**Comment by bilelmoussaoui** [comment]
Do you need a new release on elementary side to simplify things?

File: README.md [hunk 2]
```diff
@@ -48,6 +48,50 @@ jobs:
...
+    # Docker is required by the docker/setup-qemu-action which enables emulation
+    - name: Install deps
+      run: |
+        dnf -y install docker
+    - name: Set up QEMU
+      id: qemu
+      uses: docker/setup-qemu-action@v1
+      with:
+        platforms: arm64
+    - uses: bilelmoussaoui/flatpak-github-actions/flatpak-builder@v4
```
Comment (bilelmoussaoui) [inline README.md:88]: v4 doesn't exist yet, I would suggest bumping others to v4 as well and will tag a release just after it.

**Comment by bilelmoussaoui** [comment]
Thanks a lot. I will do a release later today

**Comment by davidmhewitt** [comment]
Could you let me know once you've made the release and I'll test it out with a few elementary applications? Thank you!

**Pull request merged**